scss/atoms/typography/_typography.scss

```diff
@@ -5,3 +5,120 @@
 @import
 	"variables",
 	"functions";
+
+// TODO probably should do the import in the `head` in case character sets need to change dynamically
+@import url( http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700,400italic,700italic );
+
+body {
+	font: $normal #{ map-get( $root-font, font-size ) }/1.5 $sans-serif;
+
+	@media ( max-width: 800px ) {
+		font-size: 16px;
+	}
+	@media ( max-width: 500px ) {
+		font-size: 14px;
+	}
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+	font-weight: 600;
+	line-height: 1;
+}
+
+h1 {
+	font-size: em( 67px ); // 54px works nicely, as well
+	font-weight: 700;
+}
+
+h2 {
+	font-size: em( 36px );
+}
+
+h3 {
+	font-size: em( 30px );
+}
+
+h4 {
+	font-size: em( 24px );
+}
+
+h5 {
+	font-size: em( 20px );
+}
+
+h6 {
+	font-size: em( 16px );
+	text-transform: uppercase;
+}
+
+// ==========================================================================
+// Block elements
+// ==========================================================================
+
+hr {
+	border: 0;
+	border-bottom: 1px solid #999;
+}
+
+blockquote {
+	margin: 0;
+	padding-left: 1em;
+	position: relative;
+	border-left: 4px solid #eee;
+	font-style: italic;
+}
+
+pre {
+	padding: 16px 20px;
+	background: #f7f7f7;
+	font: normal 12px/1.4 $monospace;
+}
+
+// ==========================================================================
+// Lists
+// ==========================================================================
+
+dt {
+	font-weight: 600;
+}
+
+dd {
+	margin: 0;
+}
+
+// ==========================================================================
+// Inline elements
+// ==========================================================================
+
+a {
+	color: inherit;
+	text-decoration: none;
+	background-image: linear-gradient( to bottom, rgba( 0, 0, 0, 0 ) 75%, #000 50% );
+	background-repeat: repeat-x;
+	background-size: 2px 2px;
+	background-position: 0 em( 22px );
+	outline: 0;
```

Comment:
Just to give some context on why i don't think we should alter native. We used to have non native focus styles in jQuery UI and had TONS of complaints, until we removed it.
Based of the accessibility concerns involved with focus indication i think its best to leave the style users are familiar with on their particular device. 

I do however also think we should provide a visual focus style for certain situations like showing a focus indication where native can't an example is toolbars or accessible styled radio buttons and checkboxes. This situation came up recently and after consulting with several accessibility experts this is what we came up with. 
http://view.jqueryui.com/button-classes/demos/checkboxradio/default.html
You can see tabbing a visual focus outline ( that may depending on your browser / os actually look native )

The problem is the visual button is a label. Labels can't receive focus they transfer to the underlying input which is accessibly hidden. So this means there is no focus outline visible. So we created a class that adds a visual focus indicator to the label. Problem here is the visual focus style and the regular focus style can be inconsistent on some platforms. 

We add our own focus style everywhere for this same reason in jQuery mobile and this solves the issue of visual inconsistency however not the issue of user complaints ( or my own person desire to keep things as simple as possible for end users, navigating through a page via keyboard, by keeping important visual indicators native, as the way people are familiar with, in style ).